### PR TITLE
Minor improvements to edge editing

### PR DESCRIPTION
**Pull request by pavelpykhtin**
Fixed issue with control point deletion being reset on selection change
Added control point insertion on right click
Added "Reset all control points" command to simplify layout editing. **Before**:
before
After:
after

### CODE REVIEW

File: packages/diagram/src/ui/top-left/TopLeftPanel.tsx [hunk 1]
```diff
@@ -127,6 +129,24 @@ const LayoutDriftNotification = (props: PopoverProps) => (
   </HoverCard>
 )
 
+const ResetControlPointsButton = () => {
+  const store = useDiagramStoreApi()
+
+  return (
+    <AnimatePresence>
```
Comment (davydkov) [inline packages/diagram/src/ui/top-left/TopLeftPanel.tsx]: `<AnimatePresence>` is redundant here, as `ResetControlPointsButton` already inside one from the line 193

File: packages/diagram/src/ui/top-left/TopLeftPanel.tsx [hunk 2]
```diff
@@ -127,6 +129,24 @@ const LayoutDriftNotification = (props: PopoverProps) => (
...
+          <Tooltip label="Reset all control points">
```
Comment (davydkov) [inline packages/diagram/src/ui/top-left/TopLeftPanel.tsx]: Better add  `{...portalProps}` to `Tooltip`

TLTR; when diagram is inside shadow root (generated react/webcomponent) - react portals also should be inside that root (see for other buttons)

File: packages/diagram/src/ui/top-left/TopLeftPanel.tsx [hunk 3]
```diff
@@ -183,6 +203,7 @@ export const TopLeftPanel = () => {
             <LayoutDriftNotification {...portalProps} />
           </m.div>
         )}
+        <ResetControlPointsButton />
```
Comment (davydkov) [inline packages/diagram/src/ui/top-left/TopLeftPanel.tsx]: Let show this button, if only
`readonly !== true && experimentalEdgeEditing === true`

Comment (pavelpykhtin) [reply packages/diagram/src/ui/top-left/TopLeftPanel.tsx]: Done

File: packages/diagram/src/ui/top-left/TopLeftPanel.tsx (hunk 2 above)
Comment (pavelpykhtin) [reply packages/diagram/src/ui/top-left/TopLeftPanel.tsx]: Done, thank you for the explanation!

File: packages/diagram/src/ui/top-left/TopLeftPanel.tsx (hunk 1 above)
Comment (pavelpykhtin) [reply packages/diagram/src/ui/top-left/TopLeftPanel.tsx]: Done

**Pull request merged**